base/sparse/sparsevector.jl

```diff
@@ -985,7 +985,6 @@ hvcat(rows::Tuple{Vararg{Int}}, xs::_TypedDenseConcatGroup{T}...) where {T} = Ba
 ### Unary Map
 
 # zero-preserving functions (z->z, nz->nz)
-conj(x::SparseVector) = SparseVector(length(x), copy(nonzeroinds(x)), conj(nonzeros(x)))
```

Comment:
if it doesn't have one, might be (and worth tracking along with other sparse broadcast perf in case it gets better someday and this stops being as much of a difference)